data/coop.ts

```diff
@@ -671,47 +671,47 @@ let INTERVIEWS_BOMBED = [
 
 const COOP_RATINGS = [
   {
-    group: "Co-op 1",
+    group: "1st Co-op",
```

Comment:
Any particular reason this group and the 6th group are called `1st Co-op` while groups 2 to 5 are just called `2nd`? Same thing with some of the other stats.

Reply:
That's a great question! I thought it looked pretty good stylistically as the x-axis label for the graphs (screenshot below) - but also open to making just the first co-op have the `Co-op` tag or adding it to every one if that seems more clear/consistent!

<img width="369" alt="Screen Shot 2023-04-22 at 9 33 13 PM" src="https://user-images.githubusercontent.com/14525856/233826312-e19a0e44-483d-41c4-bb67-75ec14f709ff.png">

Reply:
I think this looks good as it is.

Reply:
I suppose another alternative is to have a secondary label underneath the x-axis labels that says "Co-op term" similar to some of the other charts.